honeybee_energy/ventcool/simulation_control.py

```diff
@@ -0,0 +1,261 @@
+            lowrise rectangular mass, the resulting air flow and pressure simulated on
+            the building surfaces may be inaccurate. (Default: 0).
+        aspect_ratio: Aspect ratio of a rectangular footprint, defined as the ratio of
+            length of the short axis divided by the length of the long axis. This
+            parameter is required to automatically calculate wind pressure coefficients
+            for the AirflowNetwork simulation. If used for complex building geometries
+            that cannot be described as a highrise or lowrise rectangular mass, the
+            resulting air flow and pressure simulated on the building surfaces may be
+            inaccurate. (Default: 1).
+
+    Properties:
+        * vent_control_type
+        * reference_temperature
+        * reference_barometric_pressure
+        * reference_humidity_ratio
+        * building_type
+        * long_axis_angle
+        * aspect_ratio
+    """
+    __slots__ = ('_vent_control_type', '_reference_temperature',
+                 '_reference_barometric_pressure', '_reference_humidity_ratio',
+                 '_building_type', '_long_axis_angle', '_aspect_ratio', '_locked')
+    VENT_CONTROL_TYPES = ('SingleZone', 'MultiZoneWithDistribution',
+                          'MultiZoneWithoutDistribution',
+                          'MultiZoneWithDistributionOnlyDuringFanOperation')
+    BUILDING_TYPES = ('LowRise', 'HighRise')
+
+    def __init__(self, vent_control_type='SingleZone', reference_temperature=20,
+                 reference_barometric_pressure=101320, reference_humidity_ratio=0,
+                 building_type='LowRise', long_axis_angle=0, aspect_ratio=1):
+        """Initialize VentilationSimulationControl."""
+        self.vent_control_type = vent_control_type
+        self.reference_temperature = reference_temperature
+        self.reference_barometric_pressure = reference_barometric_pressure
+        self.reference_humidity_ratio = reference_humidity_ratio
+        self.building_type = building_type
+        self.long_axis_angle = long_axis_angle
+        self.aspect_ratio = aspect_ratio
+
+    @property
+    def vent_control_type(self):
+        """Get or set text indicating type of ventilation control type."""
+        return self._vent_control_type
+
+    @vent_control_type.setter
+    def vent_control_type(self, value):
+        assert value in self.VENT_CONTROL_TYPES, 'vent_control_type {} is not '\
+            'recognized.\nChoose from the following:\n{}'.format(
+                value, self.VENT_CONTROL_TYPES)
+        self._vent_control_type = value
+
+    @property
+    def reference_temperature(self):
+        """Get or set the temperature for the reference crack."""
+        return self._reference_temperature
+
+    @reference_temperature.setter
+    def reference_temperature(self, value):
+        assert isinstance(value, (int, float)), 'reference_temperature must be ' \
+            'a number. Got: {}.'.format(value)
```

Comment:
You know how to fix all of these now so you don't have to do a much of checks. You probably want to make sure this one is above absolute zero.